.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,88 @@
+name: 🐛 Bug Report
+description: Something isn't working as expected? 🤔
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Bug Report
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      description: |
+        The following prerequisites must be filled. If you are spot a bug please provide some details in [Environment](#environment)
+      options:
+        - label: If this is a question/suggestion, I have already considered discuss it on [Discord Server](https://discord.gg/FQ5vgW3)
+        - label: I have attempted to [look for similar issues](https://github.com/twitch4j/twitch4j/issues) already
+          required: true
+        - label: The bug have a spotted
```

Comment:
The meaning of this is unclear

Reply:
Author must know if his issue have been spotted in specific environment and providing console output with the stacktrace. If one of them not meet requirements, the issue is `invalid` and author must provide more explanation of that bug or maintainers close the issue.

**Any suggestions to that?**